sound/soc/sof/core.c

```diff
@@ -323,28 +347,31 @@ static int sof_probe(struct platform_device *pdev)
 			 "warning: failed to initialize trace %d\n", ret);
 	}
 
+	/* vFE doesn't support pm currently */
```

Comment:
comment not needed as VFE will still call PM calls, they will just be sent to VBE as IPC which can then decide what do do. i.e. it will do nothing atm